app/models/miq_action.rb

```diff
@@ -848,6 +848,44 @@ def action_container_image_analyze(action, rec, inputs)
     rec.scan
   end
 
+  def action_container_image_annotate_deny_execution(action, rec, inputs)
+    unless rec.kind_of?(ContainerImage)
+      MiqPolicy.logger.error("MIQ(#{__method__}): Unable to perform action [#{action.description}],"\
+                             " object [#{rec.inspect}] is not a Container Image")
```

Comment:
the rec.inspect is for a case where the rec is not a container image, so I'm not sure what to print